admin_guide/garbage_collection.adoc

```diff
@@ -70,6 +70,29 @@ kubeletArguments:
 ----
 ====
 
+=== Detection of containers for deletion
+
+At the moment only docker and rkt are supported. Rkt has its own GC. The following is only docker related.
```

Comment:
I think this should be a note that only shows up for Origin, and not in the OSE build. If that's fine, I can do that in my follow-up PR.